### ISSUE TITLE
pre-MVP Quiz Testing

### ISSUE THREAD

**Issue by elPiablo** [Developer-DAO/academy-turbo#184]
## Test Status of Quizzes on staging application
(I'm including more than bugs, so I'll give some positive points to give more context)
### Desktop
- [x] Brave browser with Zerion
- [ ] Result to resolve: 
-  The text 'Send Message' isn't easy to see:
![cannotSeeSendMessage](https://github.com/Developer-DAO/academy-turbo/assets/40469149/6c7d88c9-f4d9-4cce-8196-5c4ff1d184eb)
- on my first test round, I connected wallet, but didn't see the 'Send Message' button and continued to go do the quiz. 
-  I clicked on Submit with all correct answers, the spinning wheel appears, but no pop-up message appears. 
- After realising, I went back and I ***did*** sign the wallet message
-  went back to quiz and clicked on Quiz button and I found my answers were stored (in the browser cache?). 
- So went back and I passed the quiz and get  a 'congratulations, you passed this lesson' pop-up DID appear  
- when I click on 'QUIZ COMPLETED' button, I get the 'you're doing great' happy modal

- Submit button does not respond (no click) if I haven't answered all the questions 😄 
- After clicking on Submit, a red pop-ups shows 'Quiz failed You have 1 wrong answers' as expected. 😄 
- red pop-up persists (doesn't disappear) 😢 I think we need to make this a) disappear quicker or b) (more complicatied) disappear when another event happens, or button is clicked
---
- now i can see the 'send message'.....wtf? Maybe all these new browser profiles have light and dark mode set (they are all Brave profiles so far)

-
![why can see send message now](https://github.com/Developer-DAO/academy-turbo/assets/40469149/f0dc2da5-b5a8-4ac5-aded-de73c4649e45)

---

- [x] Brave browser with Metamask
- [x] Result to resolve: no issues to resolve. All good and as expected 😄 
---

- [x] Firefox browser with Zerion (Zerion not available on Firefox)
- [x] Result to resolve: n/a (= not applicable)
---
- [x] Firefox browser with Metamask
- [x] Result to resolve: no issues to resolve. All good and as expected 😄 
---
### Mobile Android
- [x] Brave browser and Zerion app
- [ ] Result to resolve: 
- modal doesn't fit well in browser window frame => result is we can't see the quiz buttons at the bottom if there is a lot of questions/answers text, therefore we can't navigate in the quiz
- using Zerion for the a) log in and b) sign in UX is ***not so*** great because we need to manually direct ourselves back to the Aca app (with MM, this happens automatically) 😢  
---
- [x] Brave browser and  Metamask app
- [ ] Result to resolve: 
- modal doesn't fit well in browser window frame => result is we can't see the quiz buttons at the bottom if there is a lot of questions/answers text, therefore we can't navigate in the quiz 😢 
- using Metamask for the a) log in and b) sign in UX is great because we are automatically directed back to the Aca app (we don't need to manually switch windows to get back) 😄 
---
- [x] Firefox browser with Zerion 
- [ ] Result to resolve: 
- after 'Connect a Wallet' stage, we leave the Firefox and log into Zerion with passcode
- we get the a chance to 'Connect' to Academy staging app, but need to manually switch windows back to Academy
- click on Academy 'Send message' button, leave Firefox app again and choose Zerion as wallet preference
- switch manually back to Academy and get 'Error signing message, please retry!' message
- in summary, we can't log in going in a loop because we are leaving the Academy app 😢
- end of test  

---
- [x] Firefox browser with Metamask
- [ ] Result to resolve: 
20.15 UTC, 16 February:
I can't activate any wallet options from Firefox - maybe some changes have just been done on staging branch

**Comment by elPiablo** [comment]
I think we can close this issue, @Markkos89, can't we?

**Comment by Markkos89** [comment]
> I think we can close this issue, @Markkos89, can't we?

indeed, thanks piablo!
